service/src/java/org/apache/hive/service/cli/operation/Operation.java

```diff
@@ -237,7 +238,9 @@ protected void createOperationLog() {
    * Set up some preconditions, or configurations.
    */
   protected void beforeRun() {
-    ShimLoader.getHadoopShims().setHadoopQueryContext(queryState.getQueryId());
+    CallerContext.setCurrent(new CallerContext.Builder("Check").build());
```

Comment:
Should this line be removed? Unless I'm mistaken, the call to `setHadoopQueryContext` on the next line will overwrite the value set by this line.

Reply:
Aah, my bad. Came through my initial dev testing & I forgot to remove it :-(